Dappfile

```diff
@@ -1,34 +1,9 @@
-name: maker-otc
-version: 0.3.1
-layout:
-  sol_sources: ./contracts
-  build_dir: ./build
-  packages_directory: ./dapple_packages
-dependencies:
-  erc20: 1.0.0
-environments:
-  develop:
-    objects: {}
-    type: TestRPC
-    chain: ''
-  live:
-    objects:
-      otc:
-        value: '0xa1b5eedc73a978d181d1ea322ba20f0474bb2a25'
```

Comment:
`maker-otc` is included as submodule by `maker-marker`, as part of the `gulp` deployment cycle, it does a `dapple build` to generate javascript files, which do include the contract addresses for mainnet and ropsten where maker-otc is deployed.
see https://github.com/makerdao/maker-market/blob/master/gulpfile.js